clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/id/DcosSpinnakerAppId.groovy

```diff
@@ -45,10 +49,41 @@ class DcosSpinnakerAppId {
      *         Deemed safe because backslashes are replaced with underscores.
      *         <p/>
      *         Ex: {@code acct_foo_bar}
-     * @see #getSafeRegion()
+     * @see #getSafeGroup()
+     */
+    public String getSafeCombinedGroup() {
+        unsafeCombinedGroup.replaceAll(MarathonPathId.PART_SEPARATOR, SAFE_REGION_SEPARATOR)
+    }
+
+    /**
+     * @return The canonical DC/OS "region" (a.k.a the full group path in which the marathon application lives)
```

Comment:
We can't remember why, and considering how far we got with this before the question came up, we felt it was probably best to just go ahead with this and then at a later date remove that if it was possible (since we may run into the reason why we do it when trying to make those changes). We didn't log a JIRA afaik though.